vrpn_Tracker_WiimoteHead.C

```diff
@@ -146,8 +146,8 @@ vrpn_Tracker_WiimoteHead::~vrpn_Tracker_WiimoteHead(void) {
 	}
 
 	// Turn off the callback handler
-	int	ret;
-	ret = d_ana->unregister_change_handler(this, handle_analog_update);
+	int	ret = d_ana->unregister_change_handler(this, handle_analog_update);
+	(void)ret;
```

Comment:
I didn't know about this cool trick to use a variable without using it.

Maybe we just remove the variable altogether and ignore the return value from the function call?

Reply:
yeah, that's probably good.

The whole cast-to-void thing looks like so much of a hack, but it's been the way to do this for a long time I think. There's a newer "possibly_unused" attribute but I haven't seen any advantage to using it. Though, this could be wrapped in a macro:

```c++
#define VRPN_POSSIBLY_UNUSED(VAR) (void)(VAR)

VRPN_POSSIBLY_UNUSED(ret);
```